typescript/packages/reality-capture-analysis/package.json

```diff
@@ -28,19 +28,15 @@
 	},
 	"license": "MIT",
 	"dependencies": {
-		"axios": "^1.6.0"
-	},
-	"peerDependencies": {
-		"@itwin/core-bentley": "^4.0.0",
-		"@itwin/core-common": "^4.0.0"
+		"axios": "^1.6.0",
+		"@itwin/reality-capture-common": "workspace:*",
```

Comment:
if only used for types should be dev dep

Reply:
the enum value is only used for tests, so i think its safe to only be a dev dep

Reply:
Fixed